backend/java/main/DBRecordModel.java

```diff
@@ -0,0 +1,27 @@
+package main;
+public class DBRecordModel {
```

Comment:
DBRecordModel doesn't say what is the model.
If you create User class, you don't call it DBUserModel :) 
Just saying...

Reply:
Renamed to User.